apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -2439,6 +2569,27 @@ get_oracle_response(Data) ->
     Host = external_address(),
     http_request(Host, post, "tx/oracle/response", Data).
 
+
+get_name_preclaim(Data) ->
+    Host = external_address(),
+    http_request(Host, post, "tx/name/preclaim", Data).
+
+get_name_claim(Data) ->
+    Host = external_address(),
+    http_request(Host, post, "tx/name/claim", Data).
+
+get_name_update(Data) ->
+    Host = external_address(),
+    http_request(Host, post, "tx/name/update", Data).
+
+get_name_transfer(Data) ->
+    Host = external_address(),
+    http_request(Host, post, "tx/name/transfer", Data).
+
+get_name_revoke(Data) ->
```

Comment:
Why do we call these `get_`? Isn't `post_` a bit more natural?

Reply:
the POST method is used out of neccesity. In fact we're *getting* the unsigned transaction (from a code point of view). Probably it is arguably which is better :(